package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "money-math",
-  "version": "2.4.5",
```

Comment:
Adding new functions to the public API isn't a patch-level change; but should rather be a minor version bump.